src/components/pager/pager.spec.js

```diff
@@ -55,6 +60,69 @@ describe('Pager', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  describe('callbacks work as expected', () => {
+    let wrapper,
+        onNext,
+        onFirst,
+        onPrevious,
+        onLast,
+        callbacks;
+
+    beforeEach(() => {
+      onNext = jest.fn();
+      onFirst = jest.fn();
+      onPrevious = jest.fn();
+      onLast = jest.fn();
+
+      wrapper = render({
+        ...props,
+        theme: mintTheme,
```

Comment:
I think that we no longer need this prop

Reply:
fixed